torch_em/data/datasets/bcss.py

```diff
@@ -82,12 +146,12 @@ def get_bcss_dataset(path, patch_shape, download=False, label_dtype=torch.int64,
 
 
 def get_bcss_loader(
-        path, patch_shape, batch_size, download=False, label_dtype=torch.int64, **kwargs
+        path, patch_shape, batch_size, split, val_fraction=0.2, download=False, label_dtype=torch.int64, **kwargs
```

Comment:
Would it be a lot of effort for you to make `split` optional, and if it's not passed return all data?

Reply:
Done ;)